chapter-06/integration-tests/src/main.rs

```diff
@@ -0,0 +1,76 @@
+use serde::{Deserialize, Serialize};
```

Comment:
`Serialize` is not used it seems...

```suggestion
use serde::Deserialize;
```